glide.yaml

```diff
@@ -26,6 +26,9 @@ import:
   subpackages:
   - proto
 
+- package: github.com/gogo/protobuf
+  version: ^v1.1.1
```

Comment:
Perhaps just pin to `v1`?  Probably are ok with any version as long as they're not breaking changes (which should be captured by a major version bump)?  

Dep is complaining more and more about conflict mismatches (now working on a few projects that are using dep instead of glide and it's a definite issue).

Reply:
sg